src/Microsoft.SqlTools.ServiceLayer/DacFx/DacFxService.cs

```diff
@@ -286,6 +287,30 @@ public async Task HandleGenerateTSqlModelRequest(GenerateTSqlModelParams request
             }
         }
 
+        /// <summary>
+        /// Handles request to get objects from sql model
+        /// </summary>
+        /// <returns></returns>
+        public async Task HandleGetObjectsFromTSqlModelRequest(GetObjectsFromTSqlModelParams requestParams, RequestContext<TSqlObjectInfo[]> requestContext)
+        {
+            try
```

Comment:
You don't need the wrapping try/catch anymore. I fixed it so that any errors are caught by the dispatcher and sent as errors to the request context automatically.